docs/dependency-specification.md

```diff
@@ -77,10 +77,25 @@ You can specify the exact version of a package.
 This will tell Poetry to install this version and this version only.
 If other dependencies require a different version, the solver will ultimately fail and abort any install or update procedures.
 
-#### Multiple requirements
+### Multiple requirements
 
 Multiple version requirements can also be separated with a comma, e.g. `>= 1.2, < 1.5`.
 
+### `@` operator
+
+You can specify requirements, using `@` operator. This is understood the same way as `==` specifier, with one difference.
+After `@` symbol, you can use any specifiers that are valid in `pyproject.toml` file. For example:
```

Comment:
After the '@' symbol, maybe.

Also, maybe best to stick with operator or symbol throughout.